src/core/features/course/components/course-format/course-format.ts

```diff
@@ -800,6 +808,10 @@ export class CoreCourseFormatComponent implements OnInit, OnChanges, OnDestroy {
             this.currentSite?.getLocalSiteConfig<string>(`${COURSE_EXPANDED_SECTIONS_PREFIX}${this.course.id}`),
         );
 
+        if (expandedSections === undefined) {
+            this.accordionMultipleValue = [];
+        }
+
         // Expand all sections if not defined.
         if (expandedSections === undefined) {
```

Comment:
This empty array line can be moved to the existing if:

```suggestion
        // Expand all sections if not defined.
        if (expandedSections === undefined) {
            this.accordionMultipleValue = [];
```